### PR TITLE
Allow to modify Presto in product tests

### PR DESCRIPTION
**Pull request by kokosing**
Allow to modify Presto in product tests

### CODE REVIEW

File: presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/run-presto.sh [hunk 1]
```diff
@@ -10,6 +10,11 @@ fi
 set -x
 
 tar xf /docker/presto-server.tar.gz -C /docker
+
+for entrypoint in /docker/entrypoint.d/*; do
```
Comment (findepi) [inline presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/run-presto.sh:14]: this is a misnomer -- entrypoint should be an entrypoint (sth run before the the command)

maybe we change the the dir to `/docker/presto-init.d/` ?

**Comment by kokosing** [comment]
Thanks for comments. AC.

**Comment by kokosing** [comment]
Tested locally, it works like a charm.

File: presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/run-presto.sh [hunk 2]
```diff
@@ -10,6 +10,13 @@ fi
 set -x
 
 tar xf /docker/presto-server.tar.gz -C /docker
+
+if test -d /docker/presto-init.d ; then
```
Comment (findepi) [inline presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/run-presto.sh:14]:
```suggestion
if test -d /docker/presto-init.d; then
```

**Pull request merged**